ProcessMaker/Http/Controllers/ProcessController.php

```diff
@@ -14,10 +14,12 @@
 use ProcessMaker\Models\Screen;
 use ProcessMaker\Models\User;
 use ProcessMaker\Traits\HasControllerAddons;
+use ProcessMaker\Traits\ProjectAssetTrait;
 
 class ProcessController extends Controller
 {
     use HasControllerAddons;
+    use ProjectAssetTrait;
```

Comment:
I think using this Trait in this Controller is not necessary. Please ignore if I'm wrong!

Reply:
Your right. Its code residue. Removed!